airflow/task/task_runner/standard_task_runner.py

```diff
@@ -84,7 +84,8 @@ def _start_by_fork(self):
             try:
                 args.func(args, dag=self.dag)
                 return_code = 0
-            except Exception:
+            except Exception as task_launch_exception:
+                self.log.exception(task_launch_exception)
```

Comment:
This will show `task_launch_exception` *twice*. The message should be something else, e.g.

```python
try:
    ...
except Exception:
    self.log.exception(
        "Failed to execute job %s fo task %s",
        self._task_instance.job_id,
        self._task_instance.task_id,
    )
```

Reply:
@uranusjr  updated commit